Dockerfile

```diff
@@ -1,40 +1,20 @@
-FROM node:12 as base
-RUN mkdir -p /usr/src/app
-WORKDIR /usr/src/app
-
-FROM base as production_build
-RUN mkdir -p /usr/src/app
-WORKDIR /usr/src/app
-
-COPY package.json package-lock.json* ./
-RUN npm ci --production
-
-ARG ENV_ENVIRONMENT='development'
-ENV ENV_ENVIRONMENT=$ENV_ENVIRONMENT
-ARG ENV_SECRET='mwmBqNcTWiKxDHygMiKhwyffaKQCVoRQ'
-ENV ENV_SECRET=$ENV_SECRET
-COPY . ./
-RUN npm run prod-build
+FROM node:12-alpine as base
+RUN mkdir /voluntarily
+WORKDIR /voluntarily
 
-FROM node:12-alpine as production
+FROM base as production
+ENV NODE_ENV=production
 ENV PORT 3122
 EXPOSE 3122
-
-COPY --from=production_build /usr/src/app /voluntarily
+COPY . /voluntarily
 WORKDIR /voluntarily
 CMD ["npm", "start" ]
 
 FROM base as development
-
 ENV NODE_ENV=development
-ENV MONGOMS_DOWNLOAD_MIRROR="http://downloads.mongodb.org"
-ENV MONGOMS_VERSION="v4.0-latest"
-ENV ENV_ENVIRONMENT='development'
-ENV ENV_SECRET='mwmBqNcTWiKxDHygMiKhwyffaKQCVoRQ'
 ENV PORT 3122
 EXPOSE 3122
-
 COPY package.json package-lock.json* ./
 RUN npm install
 COPY . ./
-CMD ["npm", "run", "dev"]
```

Comment:
Might be a good idea to split the prod and dev docker file.

As this should work for production, but we may need to modify the dev docker & docker-compose more heavily.

Reply:
Yep for sure. Can do. Feel free to commit! :p